project.sh

```diff
@@ -113,8 +112,12 @@ require-file-not-empty(){
 require-committed(){
 	local file=$1
 	if file-exists "$file";then
-		git diff --quiet --exit-code "$file" ":(exclude)package-lock.json"
-		abort-on-error "uncommitted changes in $file"
+		git diff --quiet --exit-code "$file"
+		if [ $? -ne 0 ]; then
+			echo-error "uncommitted changes in $file"
+			git diff --compact-summary "$file"
```

Comment:
Show me what's not committed!

Reply:
Lint works locally, fails on CI https://app.circleci.com/pipelines/github/jesims/device-tracking/366/workflows/a5cdd980-06f3-41b8-a092-9206095b4a7f/jobs/1443

Reply:
This might cause issues with node-projects that don't have remark installed, as it will add them to the package-lock.json (which will then fail).

Options
* Add Remark and deps to each package.json
* Try and use npx (so it doesn't touch package-lock)
* Global remark install

Going to hold this PR until then